frontend/src/components/posts/PostForm.jsx

```diff
@@ -24,8 +24,9 @@ export default function PostForm({ onSubmit, onSubmitMedia }) {
       // Invoke the passed in event callback
       onSubmit({ contentText: contentText });
       // Clear the input field
-      setContentText("");
+      
```

Comment:
Could delete this lines and optionally move `// Clear the input field` above line 29